bin_tests/src/bin/crashtracker_bin_test.rs

```diff
@@ -95,11 +99,18 @@ mod unix {
         behavior.post(output_dir)?;
 
         crashtracker::begin_op(crashtracker::OpTypes::ProfilerCollectingSample)?;
-        unsafe {
-            deref_ptr(std::ptr::null_mut::<u8>());
+        match crash_typ.as_str() {
+            "null_deref" => {
+                let x = unsafe { deref_ptr(std::ptr::null_mut::<u8>()) };
+                println!("{x}");
+            }
+            "abort" => raise(Signal::SIGABRT)?,
```

Comment:
Why do the other signals get the `"sig"` prefix and `"abort"` does not?  I'd guess it's because it's more idiomatic for `abort` to get called through the libc interface, which just `raise()`s `SIGABRT` anyway--making sure it's intentional

Reply:
changed